packages/atlas/src/components/_inputs/RadioInput/RadioInput.styles.ts

```diff
@@ -1,139 +1,163 @@
 import { css } from '@emotion/react'
 import styled from '@emotion/styled'
 
-import { cVar, transitions } from '@/styles'
+import { cVar, square } from '@/styles'
 
-export type RadioButtonStyleProps = Partial<{
-  disabled: boolean
-  checked: boolean
-}> &
-  RadioButtonCaptionProps
-
-type RadioButtonCaptionProps = {
+type StyledRadioInputProps = {
   error?: boolean
+  disabled?: boolean
 }
 
-export const Input = styled.input`
-  margin: auto;
-  opacity: 0;
-  cursor: ${(props) => (props.disabled ? 'not-allowed' : 'pointer')};
-`
+export const StyledRadioInput = styled.div<StyledRadioInputProps>`
+  ${square('16px')};
 
-export type CustomRadioInputProps = {
-  disabled?: boolean
-  checked?: boolean
-  error?: boolean
-}
+  position: relative;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-clip: content-box;
+  padding: 4px;
+  box-sizing: border-box;
+  border: 1px solid ${({ error }) => cVar(error ? 'colorBorderError' : 'colorBorderAlpha')};
+  transition: all ${cVar('animationTransitionFast')};
+
+  ::before {
```

Comment:
Could you add a comment in the code that describes what `:before` and `:after` is? Just add something like that:
```tsx
/* inner dot */ 
:after
```